YAXLib/MemberWrapper.cs

```diff
@@ -898,6 +900,10 @@ private void ProcessYaxAttribute(object attr)
             {
                 IsAttributedAsDontSerializeIfNull = true;
             }
+            else if (attr is YAXElementOrder)
+            {
+                Order = (attr as YAXElementOrder).Order;

```

Comment:
I followed the pattern established in the method.

Optimize it as you see fit.